docker/docker-compose.infra.yml

```diff
@@ -24,9 +22,9 @@ services:
   mysql-for-e2e:
     hostname: mysql
     image: mysql:8.4.0
-    platform: linux/amd64
+    platform: linux/arm64/v8
     restart: always
-    command: --default-authentication-plugin=mysql_native_password
+    command: [--mysql-native-password=ON]
```

Comment:
We didn't notice that the command should be changed as mysql version is updated.
Thanks for your fix.